sql/updates/post-update.sql

```diff
@@ -1,67 +1,107 @@
 -- needed post 1.7.0 to fixup continuous aggregates created in 1.7.0 ---
-DO $$
+CREATE OR REPLACE PROCEDURE _timescaledb_internal.fixup_caggs() AS
+$$
```

Comment:
i think we shouldnt create functions in _timescaledb_internal even if only temporary

Reply:
I know thanks... will change it!!!

Reply:
Fixed!